scalalib/src/ScalaModule.scala

```diff
@@ -185,6 +201,11 @@ trait ScalaModule extends JavaModule { outer =>
       )
   }
 
+  override def docSources: T[Seq[PathRef]] = T {
```

Comment:
Here, you generate a `Seq` of source files instead of source directories.

Reply:
Fixed in https://github.com/com-lihaoyi/mill/pull/1385/commits/e7934292cf2b2bfc781d788e1e3a530d90dc7b46